test/boost/cql_query_test.cc

```diff
@@ -3196,6 +3196,14 @@ SEASTAR_TEST_CASE(test_static_multi_cell_static_lists_with_ckey) {
     });
 }
 
+// std::pow() is not constexpr
+static constexpr int constexpr_int_pow(int x, int y) {
```

Comment:
Should be "unsigned int y" - your implementation is (for obvious reasons) incorrect for negative y.

Reply:
ok